WebRoot/js/zimbraMail/share/model/ZmZimbraAccount.js

```diff
@@ -110,7 +110,7 @@ function(name) {
 	var identity = this.getIdentity();
 	// TODO: If no identity and name is set, should create one!
 	if (!identity) return;
-	identity.name = name;
+	identity.name = AjxStringUtil.htmlEncode(name);
```

Comment:
we should do html encoding only in the view layer, data should never be modified

Reply:
but i think the the svg onload is getting  executed while sending requests

Reply:
updated